public/index.html

```diff
@@ -7,7 +7,7 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="No fees, no frills eth/weth exchange. Built by Raid Guild."
+      content="Web site created using create-react-app"
```

Comment:
Public description of website. important for seo

Reply:
updated